umap/static/umap/js/umap.forms.js

```diff
@@ -1033,6 +1033,7 @@ U.FormBuilder = L.FormBuilder.extend({
   setter: function (field, value) {
     L.FormBuilder.prototype.setter.call(this, field, value)
     if (this.options.makeDirty !== false) this.obj.isDirty = true
+    if ('render' in this.obj) this.obj.render([field], this)
```

Comment:
this may call render on map when searching in the browser. So either this is only done when `makeDirty` is false, either we use a dedicated Builder class for the browser (simpler imho).

Reply:
I don't think we should rely on `makeDirty` as I think it will be out of the picture at some point.

Using a dedicated browser class seem the way to go :-)

Reply:
In the end, I did the opposite: this call only happens if `makeDirty` is not explicitely set to `false`.

From where I stand right now, it seems to avoid adding complexity for no almost no benefits. We can take care of doing things differently if/when we remove the `dirty` mechanism.